var/spack/repos/builtin/packages/py-flit-core/package.py

```diff
@@ -33,6 +35,10 @@ class PyFlitCore(PythonPackage):
     version("3.0.0", sha256="a465052057e2d6d957e6850e9915245adedfc4fd0dd5737d0791bf3132417c2d")
     version("2.3.0", sha256="a50bcd8bf5785e3a7d95434244f30ba693e794c5204ac1ee908fc07c4acdbf80")
 
+    with default_args(type=("build", "run")):
+        depends_on("python@3.6:", when="@3.6:")
```

Comment:
Haven't checked but does the dependency on this version of python really begin with this package's version `3.6`?